### PR TITLE
[BPK-1611] Add image to the web phone input

### PR DESCRIPTION
**Pull request by tiagoengel**

### CODE REVIEW

File: packages/bpk-component-select/readme.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ export default () => (
 | onChange                  | func       | true     | -             |
 | valid                     | bool       | false    | null          |
 | large                     | bool       | false    | false         |
+| disable                   | bool       | false    | false         |
```
Comment (shaundon) [inline packages/bpk-component-select/readme.md]: typo - should say `disabled`.

File: packages/bpk-component-select/src/BpkSelect.js [hunk 2]
```diff
@@ -30,6 +30,7 @@ export type Props = {
   name: string,
   value: string,
   className: ?string,
+  disabled: ?boolean,
```
Comment (shaundon) [inline packages/bpk-component-select/src/BpkSelect.js]: I think this should just be `boolean`, as it isn't possible for it to be `null` due to having a defaultProp of `false`. Same as with `docked` etc.

**Comment by tiagoengel** [comment]
We should've done it in the select card. I'm not really adding the disabled property, the select already accepts and styles the 'disabled' version (https://github.com/Skyscanner/backpack/blob/master/packages/bpk-mixins/src/mixins/_forms.scss#L325). 

The problem is that the image was not being correctly styled (as it has now a wrapper outside the original select).

**Comment by shaundon** [comment]
@tiagoengel gotcha, thanks for clarifying 👍

**Comment by tiagoengel** [comment]
Oh right, I meant to do that but forgot 🤦‍♂️

File: packages/bpk-component-phone-input/readme.md [hunk 3]
```diff
@@ -13,6 +13,26 @@ npm install bpk-component-phone-input --save-dev
...
+import BpkImage from 'bpk-component-image';
+
+const DIALING_CODE_TO_ID_MAP = {
+  '44': 'uk',
+  '55': 'br',
+};
+
+const getFlag = dialingCode => {
+  const countryCode = DIALING_CODE_TO_ID_MAP[dialingCode];
+  const url = `https://images.skyscnr.com/images/country/flag/header/${countryCode}.png`;
```
[the remark this answers is not in the document]
Comment (tiagoengel) [reply packages/bpk-component-phone-input/readme.md]: done

**Pull request merged**